doc/design/vic-machine/configure.md

```diff
@@ -170,13 +169,33 @@ The Configure Rollback is talking about after a successful configure, vic-machin
 
 This feature is nice to have, but as vic-machine upgrade already have this capability, mark it as P0.5.
 
+# Inspect Command Enhancement
+
+- Dump certificates
```

Comment:
This is needed as part of `--conf` but I don't think there's any value in having it separate. The VCH doesn't store the certificates needed by the client.